cpp/src/io/json/reader_impl.cu

```diff
@@ -517,17 +517,23 @@ void reader::impl::set_data_types(cudaStream_t stream)
       stream);
 
     auto get_type_id = [&](auto const &cinfo) {
+      cudf::size_type int_count_total =
```

Comment:
Is it safe for this to be `cudf::size_type` here? I noticed it's an `unsigned long long` in `io/csv/reader_impl.cu`.
```suggestion
      unsigned long long int_count_total =
```

Reply:
I think `cudf::size_type` is right here, since the total will never be larger than the number of rows. I'm also okay with using `auto` here.